pysat/tests/classes/cls_instrument_library.py

```diff
@@ -198,6 +198,21 @@ def teardown_class(self):
         del self.saved_path, self.tempdir
         return
 
+    def setup_method(self):
+        """Initialize parameters before each method."""
+
+        return
+
+    def teardown_method(self):
+        """Clean up any instruments that were initialized."""
+
+        if hasattr(self, "test_inst"):
+            del self.test_inst
+        if hasattr(self, "date"):
+            del self.date
```

Comment:
```suggestion
    def setup_method(self):
        """Initialize parameters before each method."""
        self.test_inst = None
        self.date = None

        return

    def teardown_method(self):
        """Clean up any instruments that were initialized."""

        del self.test_inst, self.date
```

Alternative implementation